tests/test_input.py

```diff
@@ -1808,7 +1843,7 @@ def test_learning_curve_no_metrics():
                                          values_to_fill_dict,
                                          'learning_curve_default_objectives_and_no_metrics')
 
-    yield check_config_parsing_value_error, config_path
```

Comment:
Why did you remove the `yield` here?

Reply:
I think I deleted it :) lemme put yield back.